psalm.xml

```diff
@@ -1,6 +1,5 @@
 <?xml version="1.0"?>
 <psalm
-  totallyTyped="true"
```

Comment:
To be equivalent, we should set [`errorLevel`](https://psalm.dev/docs/running_psalm/configuration/#errorlevel) to `1`.

Reply:
Done.